src/Console/Command/PullRequestAutoMergeCommand.php

```diff
@@ -0,0 +1,135 @@
+    {
+        foreach ($this->configs['projects'] as $name => $projectConfig) {
+            try {
+                $package = $this->packagistClient->get(static::PACKAGIST_GROUP.'/'.$name);
+                $this->io->title($package->getName());
+                $this->mergePullRequest($package, $projectConfig);
+            } catch (ExceptionInterface $e) {
+                $this->io->error('Failed with message: '.$e->getMessage());
+            }
+        }
+
+        return 0;
+    }
+
+    private function mergePullRequest(Package $package, array $projectConfig)
+    {
+        if (!array_key_exists('branches', $projectConfig)) {
+            return;
+        }
+
+        $repositoryName = $this->getRepositoryName($package);
+        $branches = array_keys($projectConfig['branches']);
+
+        $pulls = $this->githubPaginator->fetchAll($this->githubClient->pullRequests(), 'all', array(
+            static::GITHUB_GROUP,
+            $repositoryName,
+        ));
+        foreach ($pulls as $pull) {
+            // Do not managed not configured branches.
+            if (!in_array($pull['base']['ref'], $branches, true)) {
+                continue;
+            }
+
+            // Proceed only bot PR for now.
+            if ('SonataCI' !== $pull['user']['login']) {
+                continue;
+            }
+
+            $this->io->section(sprintf(
+                '#%d > %s - %s',
+                $pull['number'],
+                $pull['base']['ref'],
+                $pull['title']
+            ));
+
+            $state = $this->githubClient->repos()->statuses()->combined(
+                static::GITHUB_GROUP,
+                $repositoryName,
+                $pull['head']['sha']
+            );
+
+            $this->io->comment('Author: '.$pull['user']['login']);
+            $this->io->comment('Branch: '.$pull['base']['ref']);
+            $this->io->comment('Status: '.$state['state']);
+
+            // Ignore the PR is status is not good.
+            if ('success' !== $state['state']) {
+                continue;
+            }
+
+            $commits = $this->githubPaginator->fetchAll($this->githubClient->pullRequests(), 'commits', array(
+                static::GITHUB_GROUP,
+                $repositoryName,
+                $pull['number'],
+            ));
+
+            $commitMessages = array_map(function ($commit) {
+                return $commit['commit']['message'];
+            }, $commits);
+            $commitsCount = count($commitMessages);
+            $uniqueCommitsCount = count(array_unique($commitMessages));
+
+            // Some commit have the same message, but this cannot be squashed.
+            if ($commitsCount !== $uniqueCommitsCount && 1 !== $uniqueCommitsCount) {
+                $this->io->caution('This PR need a manual rebase.');
+                continue;
+            }
+            $squash = 1 === $uniqueCommitsCount ? true : false;
```

Comment:
`1 === $uniqueCommitsCount` can only return a boolean, so this ternary is useless? You might as well do this :trollface: 

```php
<?php
$squash = 1 === ($uniqueCommitsCount ? true : false) : true : false;
```